application/dash_application/views/runreport.py

```diff
@@ -12,12 +12,7 @@
 
 page_name = "runreport/proj_hist"
 
-ids = init_ids([
-    'project',
-    'focused_run',
-    'coverage_dist',
-    'click-data'
-])
+ids = init_ids(["project", "focused_run", "coverage_dist", "click-data"])
```

Comment:
Here too, I find having all the items on one line harder to read

Reply:
updated